reports/post_diagnostics.Rmd

```diff
@@ -47,6 +57,65 @@ kable(log_status %>%
 
 ---
 
+# Dashboard update frequency
+
+**How many days has it been since we saw data updates from each scraped dashboard?** 
+
+```{r update-frequency, echo = F, message = F, warning = F, fig.height = 10, fig.width = 15}
+last_df <- pull_last_update()
+last_df %>%
+    filter(!is.na(days_late)) %>%
+    # only get these kind of scrapers
+    filter(jurisdiction %in% c("state", "federal", "immigration")) %>%
+    # remove population scrapers here
+    filter(!stringr::str_detect(id, "(?i)population")) %>%
+    # remove youth scrapers here
+    filter(!stringr::str_detect(id, "(?i)youth")) %>%
```

Comment:
Not the point of this PR, but I'm wondering if removing `(?i)vaccine` scrapers here might make sense! (I could go either way tbh)

Reply:
I think I could go either way too! I'm going to leave the vaccine scrapers in for now, but if we want to remove them down the road it's easy enough to change